src/main/scala/net/snowflake/spark/snowflake/io/package.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2018 Snowflake Computing
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.snowflake.spark.snowflake
+
+import net.snowflake.spark.snowflake.Parameters.MergedParameters
+import net.snowflake.spark.snowflake.io.SupportedFormat.SupportedFormat
+import net.snowflake.spark.snowflake.io.SupportedSource.SupportedSource
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.types.StructType
+import org.apache.spark.sql.{SQLContext, SaveMode}
+
+/**
+  * Interface to IO component
+  */
+package object io {
```

Comment:
Capital I.

Reply:
I can't change it. This is a package object, the name should exactly match the name of package. If I change this name, I have to change the package name to IO also.